packages/api/src/webhooks/cannon.ts

```diff
@@ -69,7 +134,14 @@ export default class WebhookCannon {
       return;
     }
     try {
-      await this.onTrigger(message);
+      // TODO Activate URL Verification
+      await this._fireHook(
+        message.event,
```

Comment:
Can we check that `event`, `webhook`, `stream`, `user` fields exists in `message` object? Just to check that we got valid data.